generics/messageConstants/common.js

```diff
@@ -54,5 +54,8 @@ module.exports = {
     CREATED_BY_ME : "createdByMe",
     ALL_ROLES : "all",
     VERSION_3 : "v3",
-    DRAFT : "draft"
+    DRAFT : "draft",
+    OK : "OK",
+    SUNBIRD_SERVER_TIMEOUT : 5000,
```

Comment:
@VISHNUDAS-tunerlabs move this 2 keys in env and if it is missing in env define the value in the top of that particular file. remove it from common file